README.md

```diff
@@ -0,0 +1,79 @@
+# Upcoming Movies Web App
+This is a MVP project to list upcoming movies integrate with [The Movie Database](https://themoviedb.org).
+
+# Libraries used!
+  - [ReactJS](https://reactjs.org/)
+    * [Create React App](https://github.com/facebook/create-react-app)
+  - [Redux](https://redux.js.org/)
+  - [Lumen](https://lumen.laravel.com/)
+    * [Guzzle](https://github.com/guzzle/guzzle)
+ 
+# Architecture
+This solution was split into Front and Backend.
+The **business layer** was made using *Lumen* and *MVC* architecture to be able to separate the responsabilities, once it was necessary to abstract the access to the data, as they are being provided by **TMDb**. To get the data was used *Guzzle PHP Http Client*.
+In order to improve the User Experience was used caching techniques to not overload the *TMDb Service.*
```

Comment:
In onder to improve the User Experience, **it was used** to **don't overload** ...